test/test_sbol2_sbol3_direct.py

```diff
@@ -1,6 +1,9 @@
+from sbol2 import Config, ConfigOptions
```

Comment:
sbol2 is already imported below; please use references as `sbol2.Config`, etc., rather than importing.
Also, import is out of order.